store/members/members-context.js

```diff
@@ -0,0 +1,31 @@
+import { membersReducer } from './members-reducer';
+import PropTypes from 'prop-types';
+const { createContext, useReducer, default: React, useContext } = require('react');
+
+const MembersContext = createContext({});
+const initialState = {
+  membersArr: [],
+  newMembersArr: [],
+  errorMsg: ''
+};
+
+export const MembersProvider = ({ children }) => {
+  const [state, dispatch] = useReducer(membersReducer, initialState);
+  return (
+    <MembersContext.Provider value={{ membersState: state, membersDispatch: dispatch }}>
```

Comment:
don't think so we explicitly have to specify `membersState & membersDispatch`.
Since it's in members' repo, it's well understood. What do you think?

Reply:
Yeah, I was thinking of moving ahead with having single member context too as it would help accessing role of user or with getting logged in user details 
Something I was visioning in that fashion
If we are thinking in that direction having named states would be helpful right?
Open to suggestions